CHANGELOG.md

```diff
@@ -1,8 +1,95 @@
+## 1.2.0
+
+This is a feature and improvement release.
+
+* [PCP-543](https://tickets.puppetlabs.com/browse/PCP-543) Add CA mismatch as
```

Comment:
Seems like it'll be ok; there's some weirdness going on around releasing everything.